driver-mapping/src/main/java/com/datastax/driver/mapping/Mapper.java

```diff
@@ -127,7 +134,23 @@ public MappingManager getManager() {
      * @return a query that saves {@code entity} (based on it's defined mapping).
      */
     public Statement saveQuery(T entity) {
-        PreparedStatement ps = getPreparedQuery(QueryType.SAVE);
+        return saveQuery(entity, this.defaultSaveOptions);
+    }
+
+    /**
+     * Creates a query that can be used to save the provided entity.
+     * <p>
+     * This method is useful if you want to setup a number of options (tracing,
```

Comment:
Are we planning on adding support to enable tracing and setting of CL through Options?  Currently it looks like you can only do timestamp and ttl.